<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLMemoryViewWidgetBase</class>
 <widget class="QWidget" name="KLMemoryViewWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory view</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pbRefresh">
       <property name="toolTip">
        <string>Refresh selected memory cell</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbUpdateRegularly">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update e&amp;very:</string>
       </property>
       <property name="shortcut">
        <string>Alt+V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tniUpdateRegularly">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Q3GroupBox" name="gbMemoryCell" native="true">
     <property name="title" stdset="0">
      <string>Memory cell 0x0</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line1">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel6">
          <property name="text">
           <string>Binary:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLSpinBoxWidget" name="kisBinary">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>7</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit7">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>6</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit6">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>5</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit5">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>4</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit4">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit3">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>2</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit2">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit1">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="tlBit_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbBit0">
              <property name="text">
               <string/>
              </property>
              <property name="shortcut">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tbSet">
        <property name="toolTip">
         <string>Write the value to the memory cell</string>
        </property>
        <property name="text">
         <string>W</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout">
        <item row="1" column="2">
         <widget class="QLabel" name="textLabel5">
          <property name="text">
           <string>char:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>Hex:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel4">
          <property name="text">
           <string>Dec:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leName">
          <property name="text">
           <string>r00</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="leChar">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="KLSpinBoxWidget" name="kisHex">
          <property name="prefix">
           <string>0x</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="kisDec">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="tlDescription">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="Q3ListBox" name="lbMemory" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLSpinBoxWidget</class>
   <extends>QSpinBox</extends>
   <header location="global">klspinboxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QWidget</extends>
   <header>q3groupbox.h</header>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>QWidget</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
